src/error.ts

```diff
@@ -1,7 +1,9 @@
 import chalk from 'chalk';
 import { Response } from 'node-fetch';
+import { SkygearError } from '@skygear/node-client';
 
 export const ErrorNameInvalidArgument = 'InvalidArgument';
+export const ErrorNamePermissionDenied = 'PermissionDenied';
```

Comment:
There two constants are exposed by the SDK. We do not need to redefine them here.